etcdctl/ctlv3/ctl.go

```diff
@@ -95,6 +102,27 @@ func init() {
 	)
 }
 
+func HandleConfigFile(cmd *cobra.Command) {
+	v := viper.New()
+	if globalFlags.ConfigFile != "" {
+		v.SetConfigFile(globalFlags.ConfigFile)
+	} else {
+		v.SetConfigName("config")
+		v.AddConfigPath("$HOME/.etcdctl")
+	}
+	if err := v.ReadInConfig(); err != nil {
+		// Error, unless it's just a missing file
```

Comment:
Remove this comment line? It seems to be unnecessary. The code is self-explain.

Reply:
Sure